index.html

```diff
@@ -95,6 +93,36 @@ <h2>Facebook 360</h2>
           </div>
       </section>
       <!-- First section ends -->
+      <div id="myModal" class="modal">
+        <div class="modal-container">
+          <div class="modal-content">
+            <div class="modal-title">
+              <div class="dflex">
+                <h1 id="modalTitle">Title</h1>
+
+                <span class="xp">&times;</span>
```

Comment:
- [ ] OPTIONAL
To improve the user experience it will be a nice idea that when a mouse hovers on your close button, the mouse's cursor changes to a pointer. This can be implemented by adding `cursor: pointer` to the close button's CSS attributes/properties.